velox/dwio/common/SelectiveStructColumnReader.cpp

```diff
@@ -276,6 +275,12 @@ void setConstantField(
     VectorPtr& field) {
   if (field && field->isConstantEncoding() && field.unique() &&
       field->size() > 0 && field->equalValueAt(constant.get(), 0, 0)) {
+    if (field->encoding() == VectorEncoding::Simple::ROW) {
```

Comment:
The condition can never be true because we already checked `field->isConstantEncoding()` above